command/apply/internal/match/match.go

```diff
@@ -14,6 +14,9 @@ import (
 	"github.com/craftcms/nitro/pkg/containerlabels"
 )
 
+// DEFAULT_MOUNTS is the number of mounts for a standard container
+const DEFAULT_MOUNTS = 3
```

Comment:
Expanding on what @angrybrad was saying…
Wonder if it would be more clear to do something like:

```
const DEFAULT_MOUNTS = ["home", "certs", "nginx"]
```

Then later reference `len(DEFAULT_MOUNTS)`?

Same difference, but it is readable w/o the comments.

Reply:
Going to leave as 3, since we might change those in the future.

Reply:
👍  maybe move the comment about what the 3 mounts are up to the const?